### ISSUE TITLE
cannot import fireshape.zoo in docker image

### ISSUE THREAD

**Issue by APaganini** [fireshape/fireshape#53]
The following example run in the docker image fireshape/fireshape

```
from firedrake import *
from fireshape import *
import fireshape.zoo as fsz
```

returns

`ModuleNotFoundError: No module named 'fireshape.zoo'`

**Comment by florianwechsung** [comment]
Can you post the commands needed to reproduce this, i.e. the command you used to start the docker image and how you installed fireshape?

**Comment by APaganini** [comment]
After installing docker, make sure docker is activated, then open a terminal, and type

`docker run -ti fireshape/fireshape`

then, once in the container, activate firedrake with

`source firedrake/bin/activate`.

At this point, running (in ipython)

```
from firedrake import *
from fireshape import *
import fireshape.zoo as fsz
```

returns

`
ModuleNotFoundError: No module named 'fireshape.zoo'`


------------------------------------------------------------------------------------------------------------

Fireshape is installed as follows (see `fireshape/docker/dockerhub/Dockerfile`)

```
FROM firedrakeproject/firedrake:latest

MAINTAINER Alberto Paganini <admp1@le.ac.uk>

USER root
RUN apt-get update \
    && apt-get -y dist-upgrade \
    && apt-get -y install gmsh patchelf \
    && rm -rf /var/lib/apt/lists/*

USER firedrake
RUN . /home/firedrake/firedrake/bin/activate; pip3 install wheel --upgrade
RUN . /home/firedrake/firedrake/bin/activate; pip3 install scipy
RUN . /home/firedrake/firedrake/bin/activate; pip3 install roltrilinos
RUN . /home/firedrake/firedrake/bin/activate; pip3 install ROL 
RUN . /home/firedrake/firedrake/bin/activate; pip install git+https://github.com/fireshape/fireshape.git
```

**Comment by Ainlina** [comment]
The same issue arises for me on my local install of Fireshape; if I activate Firedrake, I can `import fireshape` without issue, but `import fireshape.zoo` fails with `ModuleNotFoundError: No module named 'fireshape.zoo'`. This causes some of the examples to fail, including `levelset_boundary,py`.

I installed Firedrake following the instructions (the "for users" installation) in the documentation at [https://fireshape.readthedocs.io/en/latest/installation.html](https://fireshape.readthedocs.io/en/latest/installation.html), on Xubuntu 20.04.3 LTS.

**Comment by APaganini** [comment]
fixed in #76
